privacyidea/static/components/config/views/config.periodictasks.details.html

```diff
@@ -0,0 +1,166 @@
+<h2 class="form-signin-heading"
+    ng-show="ptaskid">
+    <translate>Edit Periodic Task {{ ptaskid }}</translate>
+
+    <span ng-show="checkRight('periodictask_write')">
+    <button class="btn btn-danger"
+            ng-click="disablePeriodicTask(ptaskid)"
+            ng-show="form.active" translate>
+        Disable
+    </button>
+    <button class="btn btn-success"
+            ng-click="enablePeriodicTask(ptaskid)"
+            ng-show="!form.active" translate>
+        Enable
+    </button>
+    <button class="btn btn-danger"
+        ng-click="delPeriodicTask(ptaskid)">
+        <span class="glyphicon glyphicon-trash"></span>
+        <span translate>Delete</span>
+    </button>
+    </span>
+</h2>
+
+<h2 class="form-signin-heading"
+        ng-hide="ptaskid" translate>Create a new Periodic Task</h2>
+
+<form name="formPeriodicTaskAdd" role="form" validate
+      class="form-horizontal">
+
+    <div class="form-group">
+        <label for="name" class="col-sm-3 control-label" translate>
+            Description</label>
+        <div class="col-sm-6">
+            <input id="name" class="form-control"
+                   ng-model="form.name"
+                   placeholder="Name"
+                   required/>
+        </div>
+    </div>
+    <div class="form-group">
+        <label for="interval" class="col-sm-3 control-label" translate>
+            Interval</label>
+        <div class="col-sm-6">
+            <input id="interval" class="form-control"
+                   ng-model="form.interval"
+                   placeholder="Interval"
+                   required/>
+            <p class="help help-block" translate>
+                Please use crontab notation to specify the times at which the periodic task
+                should be run. Examples: "5 10 * * *" runs the task every day at 10:05,
+                and "*/10 * * * *" runs the task every ten minutes.
+            </p>
```

Comment:
What do you think of this:

~~~~html
            <p class="help help-block" translate>
                Please use crontab notation to specify the times at which the periodic task
                should be run.</p>
            <accordion>
                <accordion-group heading="{{ 'Examples'|translate}}">
                    <ul translate>
                        <li><b>"5 10 * * *"</b> runs the task every day at 10:05 am.</li>
                        <li><b>"*/10 * * * *"</b> runs the task every ten minutes.</li>
                        <li><b>"1 23 * * 7</b>runs the task at 11:01pm on Sundays.</li>
                    </ul>
                </accordion-group>
            </accordion>
~~~~
This way the user gets more information and can hide or show it.